src/components/ValidatedInput/ValidFieldText.scss

```diff
@@ -11,6 +11,12 @@
 .valid-field-text {
   position: relative;
 
+  &.password-field-text {
```

Comment:
have you checked ValidFieldText.scss in AP? i think it has more styles in it

Reply:
Yes, from the common-component in admin portal for the class `password-field-text` it only need this one styling. If you mean having more styles its actually class `.password-eye-icon`.

Reply:
yeah, i just commented in arbitrary line. the thing is that we have more styles in AP.

Reply:
I see, I put the selector for class `.password-eye-icon` in the bottom of the files,[ line 57](https://github.com/AccelByte/justice-ui-library/pull/1/files#diff-d3da3914f4a126df2a1f17b411d9856c1c68fd4dcd484c15c7aa062910723758R57)